FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h

```diff
@@ -87,6 +87,12 @@ typedef void (^FIRRemoteConfigFetchCompletion)(FIRRemoteConfigFetchStatus status
 typedef void (^FIRRemoteConfigActivateCompletion)(NSError *_Nullable error)
     NS_SWIFT_NAME(RemoteConfigActivateCompletion);
 
+/// Completion handler invoked by activate method upon completion.
+/// @param changed  Flag indicating whether or not the configuration changed.
+/// @param error  Error message on failure. Nil if activation was successful.
+typedef void (^FIRRemoteConfigActivateChangeCompletion)(BOOL changed, NSError *_Nullable error)
```

Comment:
can you illustrate a bit on what the advantage is. I'm guessing using the boolean as first parameter will improve checking in swift but I don't know what exactly it is.

Reply:
It gives feedback to the API client about whether or not the config changed without an error being thrown. See #3586 for more on the issues caused by the other API.

Reply:
Nice, so this will align with Android.

Reply:
And JS :)